syfosm-common-models/src/main/kotlin/no/nav/syfo/model/Merknad.kt

```diff
@@ -0,0 +1,6 @@
+package no.nav.syfo.model
+
+data class Merknad(
+    val type: String,
```

Comment:
Er det ikke greiere med en enum her, for å enkelt kunne se hvilket sett av strenger som er gyldig?

Reply:
Jo, egentlig er det det, men da kan vi ikke legge til verdier (da er det ikke bakoverkompatibelt) 😞